postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -319,15 +323,17 @@ def _filter_and_normalize_statement_rows(self, rows):
         normalized_rows = []
         for row in rows:
             total_count += 1
-            if not row['datname']:
+            if (not row['datname'] or not row['query']) and row.get(
+                'backend_type', 'client backend'
+            ) == 'client backend':
```

Comment:
aren't already filtering this out in our query so we should never see rows of this type?